test/TEST-56-EXIT-TYPE/test.sh

```diff
@@ -6,4 +6,9 @@ TEST_DESCRIPTION="test ExitType=cgroup"
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+if [[ "$(get_cgroup_hierarchy)" != unified ]]; then
+    echo "This test requires unified cgroup hierarchy, skipping..."
+    exit 0
```

Comment:
write to /skipped ?

Reply:
This runs outside of the test VM/container, so there's no `/{skipped,testok,failed}`, e.g.

https://github.com/systemd/systemd/blob/23b1e8d087c9e8c5a2cdcc6a91510a4e7ca8f72f/test/TEST-50-DISSECT/test.sh#L17-L19

and other.

Reply:
ah right

Reply:
we should probably concoct something, like exit 77, but another day

Reply:
Definitely. @yuwata have already been working on it in https://github.com/systemd/systemd/pull/21833, unfortunately encountered some issues along the way regarding `make`.